addons/portal/static/src/scss/portal.scss

```diff
@@ -182,6 +182,12 @@ hr {
     background-color: inherit;
 }
 
+// IE 11 hack
+// http://browserhacks.com/#hack-d19e53a0fdfba5ec0f283ae86175a3af
+*::-ms-backdrop, :root, .o_portal_sale_sidebar .flex-lg-grow-0 {
```

Comment:
Better, but since the fix it's really specific (252px), I'd prefer to affect the element only (adding a class maybe?). 

Did you already checked if it's ok on Edge?

Reply:
Work fine on Edge, the only elements in odoo using the `.o_portal_sale_sidebar` class are the sidebars affected by the bug.

Reply:
What about using a real solution like fixing the template ? `col-lg flex-lg-grow-0` makes no sense, it says "hey take the space required to fill the remaining space starting from 0 width but hey... you cannot grow !" ... using `col-lg-auto` seems to be what was wanted here and would probably work on IE11.

Reply:
> The only elements in odoo using the .o_portal_sale_sidebar class are the sidebars affected by the bug.

I'm not a big fan of specific rules targeting framework classes 😕 (more down).
Website team ? @seb-odoo , @kea14 
(@qsm-odoo is in holiday)

---------

If we add (or xpath) a new element with `.flex-lg-grow-0` class, it will be affected too. It's not an issue at the moment, but it will complicate things in the future.

Eg. _"Client complain some elements are too big"_
`[...].o_portal_sale_sidebar .flex-lg-grow-0:not(btn):not(div:first-child()):not(.bg-beta)` 😢

Moreover, it's probably the sidebar that should have the `o_portal_sale_sidebar` class (rather than the main container).
I suggest to review how classes are assigned here (solving the double classes assignment you showed me too) and then apply the fix on the right element only.

Reply:
And I am not on holiday, I am actually dying in India.

Reply:
> And I am not on holiday, I am actually dying in India.

I can feel the same heat. :sweat: